lib-ee/emqx_ee_bridge/src/kafka/emqx_bridge_impl_kafka_producer.erl

```diff
@@ -30,6 +33,7 @@ on_start(InstId, Config) ->
         authentication := Auth,
         ssl := SSL
     } = Config,
+    _ = maybe_install_wolff_telemetry_handlers(),
```

Comment:
there should be an uninstall in the `on_stop` callback?

Reply:
:heavy_check_mark: